ext/standard/tests/math/base_convert_variation2.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Test base_convert() function : strange literals
+--FILE--
+<?php
+echo 'Binary to decimal:', \PHP_EOL;
+var_dump(base_convert('0b', 2, 10));
+var_dump(base_convert('0B', 2, 10));
+var_dump(base_convert('', 2, 10));
+echo 'Octal to decimal:', \PHP_EOL;
+var_dump(base_convert('0o', 8, 10));
+var_dump(base_convert('0O', 8, 10));
+var_dump(base_convert('0', 8, 10)); // This is somehow deprecated????
```

Comment:
What's up with this comment?

Reply:
Ah right, that was when I messed up by confusing ``0`` and ``O`` ... will drop it